core/light-client/src/lib.rs

```diff
@@ -285,13 +277,14 @@ where
 					// see issue #353
 					error!("Block {:?} contained invalid justification: {:?}", block_num, err);
 					relay.unjustified_headers.push(header.hash());
-					relay.last_finalized_block_header = header;
+					relay.set_last_finalized_block_header(header);
 					return Ok(())
```

Comment:
so the name `relay` here is a `chain relay`, which was re-named to `light client` recently?

Reply:
Well, `relay` is the `RelayState` struct. Which was not yet renamed, but it would actually make sense to do that. Made an issue #432